Description: Use system specific path for shared data
Author: Balasankar C <balasankarc@autistici.org>
Last-Update: 2015-12-21
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/lib/mime/types/data.rb
+++ b/lib/mime/types/data.rb
@@ -13,7 +13,7 @@
       # System maintainer note: this is the constant to change when packaging
       # mime-types for your system. It is recommended that the path be
       # something like /usr/share/ruby/mime-types/.
-      PATH = File.expand_path('../../../../data', __FILE__)
+      PATH = File.expand_path('/usr/share/ruby-mime-types-data/data', __FILE__)
     end
   end
 end
